Example/SFFeatureFlag/FirebaseRemoteConfigService.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  FirebaseRemoteConfigService.swift
+//  SFFeatureFlag
+//
+//  Created by Maja Sapunova on 17.6.21.
+//  Copyright © 2021 CocoaPods. All rights reserved.
+//
+
+import Foundation
+import FirebaseCore
+import FirebaseRemoteConfig
+import SFFeatureFlag
+
+class FirebaseRemoteConfigService: RemoteConfigServiceProtocol {
+   
+    //MARK: - Properties
+    private var remoteConfig: RemoteConfig?
+    
+    //MARK: - RemoteConfigServiceProtocol
+    func setup() {
+        FirebaseApp.configure()
+        
+        remoteConfig = RemoteConfig.remoteConfig()
+        
+        let settings = RemoteConfigSettings()
+        
+        //WARNING: Use the minimumFetchInterval only for debug builds
+        settings.minimumFetchInterval = 0
+        
+        remoteConfig?.configSettings = settings
+        
+        setDefaults()
+    }
+    
+    func fetchRemoteConfig(_ completionHandler: @escaping RemoteConfigCompletion) {
+        RemoteConfig.remoteConfig().fetch(completionHandler: { status, error in
+            if status == .success {
+                self.remoteConfig?.activate(completion: { success, error in
+                    completionHandler(true, error)
+                })
+            } else {
+                completionHandler(false, error)
+            }
+            
+        })
+    }
+    
+    func configValue<T>(forKey key: String, type: T.Type) -> T? {
+        if type == Data.self {
+            return remoteConfig?.configValue(forKey:key).dataValue as? T
```

Comment:
I know it's nit-picking but ```(forKey:key)``` is missing space.

Reply:
Added the missing spaces